astropy/wcs/tests/test_utils.py

```diff
@@ -238,64 +238,89 @@ def test_wcs_to_celestial_frame():
     from ...coordinates.builtin_frames import ICRS, ITRS, FK5, FK4, Galactic
 
     mywcs = WCS(naxis=2)
+    mywcs.wcs.set()
     with pytest.raises(ValueError) as exc:
         assert wcs_to_celestial_frame(mywcs) is None
     assert exc.value.args[0] == "Could not determine celestial frame corresponding to the specified WCS object"
 
+    mywcs = WCS(naxis=2)
     mywcs.wcs.ctype = ['XOFFSET', 'YOFFSET']
+    mywcs.wcs.set()
     with pytest.raises(ValueError):
         assert wcs_to_celestial_frame(mywcs) is None
 
+    mywcs = WCS(naxis=2)
     mywcs.wcs.ctype = ['RA---TAN', 'DEC--TAN']
+    mywcs.wcs.set()
     frame = wcs_to_celestial_frame(mywcs)
     assert isinstance(frame, ICRS)
 
+    mywcs = WCS(naxis=2)
+    mywcs.wcs.ctype = ['RA---TAN', 'DEC--TAN']
     mywcs.wcs.equinox = 1987.
+    mywcs.wcs.set()
+    print(mywcs.to_header())
```

Comment:
Debugging leftover?

Reply:
oops, sorry I didn't notice this one when merged.

Reply:
I should have mentioned this in the "approval" as well.

Reply:
It's in the test, so maybe not a big issue?

Reply:
I agree, we'll remove it in the next PR.

Reply:
Well, it's a bit late to remove it in this PR 😂